source/NPC.cpp

```diff
@@ -69,6 +69,8 @@ void NPC::Load(const DataNode &node)
 			succeedIf |= ShipEvent::CAPTURE;
 		else if(node.Token(i) == "provoke")
 			succeedIf |= ShipEvent::PROVOKE;
+		else if(node.Token(i) == "repaired_in_bay")
```

Comment:
```suggestion
		else if(node.Token(i) == "repaired in bay")
```
underscores are not really our style?